src/components/BotKubeFeatures/index.tsx

```diff
@@ -15,7 +15,7 @@ const FeatureList: FeatureItem[] = [
     iconClass: "fas fa-user-secret",
     description: (
       <>
-        The backend for the BotKube app runs in your Kubernetes cluster - thus you have complete control on your data
+        The back end for the BotKube app runs in your Kubernetes cluster so you have complete control of your data
```

Comment:
```suggestion
        The back-end for the BotKube app runs in your Kubernetes cluster so you have complete control of your data
```

Googling around, it's mostly `backend` or `back-end` - your choice.

Reply:
I see that this command is still not applied. Should we fix it before merge? While I'm not a native, I agree with Ezo - Usually I saw `back-end` or `backend`.

Reply:
https://grammarhow.com/backend-back-end-or-back-end/

"back end" is a noun, "back-end" is an adjective, "backend" is not a word.

Reply:
Also see the canonical tech writing style guide: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/b/back-end